lab_4_fill_words_by_ngrams/main.py

```diff
@@ -125,6 +204,11 @@ def get_conversion_generator_type(self, generator_type: int) -> str:  # type: ig
         Returns:
             (str): Name of the generator.
         """
+        if generator_type == 0:
+            return 'Greedy Generator'
+        if generator_type == 1:
+            return 'Top-P Generator'
+        return 'Beam Search Generator'
```

Comment:
what if `generator_type` equels 3/4/5/6?

Reply:
changed it